src/orquestra/sdk/_base/cli/_dorq/_ui/_errors.py

```diff
@@ -87,6 +87,16 @@ def _(e: exceptions.WorkflowRunNotFinished) -> ResponseStatusCode:
     return ResponseStatusCode.INVALID_WORKFLOW_RUN
 
 
+@pretty_print_exception.register
+def _(e: exceptions.RayNotRunningError) -> ResponseStatusCode:
+    click.echo(
+        "Could not find any running Ray instance. "
+        "You can use 'orq status' to check the status of the ray service. "
+        "If it is not running, it can be started with the `orq up` command."
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** Tweaked message content, up to you!

```suggestion
        "Could not find a running Ray instance. "
        "You can use `orq status` to check the status of Ray. "
        "If it is not running, it can be started with the `orq up` command."
```